src/sas/qtgui/Calculators/Shape2SAS/DesignWindow.py

```diff
@@ -605,14 +634,11 @@ def getPluginModel(self):
         Npoints = int(self.constraint.variableTable.Npoints.text())
         prPoints = int(self.constraint.variableTable.prPoints.text())
         modelName = self.constraint.variableTable.pluginModelName.text()
-        parNames = self.getAllTableNames(self.ifNoCondition)
-        checkedPars = self.checkedVariables()
         parVals = self.getStandardReadOfTableData()
 
-        #get chosen fit parameters
-        fitPar = self.getFitParameters()
+        # Define checkedPars before using it
+        checkedPars = self.checkedVariables()
```

Comment:
Why are we getting rid of `parNames = ` and `fitPar =`?  that seems very wrong? Moving `checkedPars` from before `parVals =` might be reasonable but am guessing that forgettig to define these will have consequenses?